lib/ansible/modules/cloud/cloudscale/__init__.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
```

Comment:
`__init__.py` under modules must be empty
Shared code should be in `lib/ansible/module_utils/`

Reply:
Even if it's only shared between related modules in the same directory/python module?